dotnet/src/Connectors/Connectors.OpenAI/Core/ClientCore.TextToAudio.cs

```diff
@@ -58,15 +58,17 @@ private static GeneratedSpeechVoice GetGeneratedSpeechVoice(string? voice)
         };
 
     private static (GeneratedSpeechFormat? Format, string? MimeType) GetGeneratedSpeechFormatAndMimeType(string? format)
-        => format?.ToUpperInvariant() switch
+    {
+        switch (format?.ToUpperInvariant())
         {
-            "WAV" => (GeneratedSpeechFormat.Wav, "audio/wav"),
-            "MP3" => (GeneratedSpeechFormat.Mp3, "audio/mpeg"),
-            "OPUS" => (GeneratedSpeechFormat.Opus, "audio/opus"),
-            "FLAC" => (GeneratedSpeechFormat.Flac, "audio/flac"),
-            "AAC" => (GeneratedSpeechFormat.Aac, "audio/aac"),
-            "PCM" => (GeneratedSpeechFormat.Pcm, "audio/l16"),
-            null => (null, null),
-            _ => throw new NotSupportedException($"The format '{format}' is not supported.")
-        };
+            case "WAV": return (GeneratedSpeechFormat.Wav, "audio/wav");
```

Comment:
Just wondering what is the purpose of this change?

Reply:
The latest change in `OpenAI SDK` breaks usage with switch expression in the line 69 (ArgumentNull Exception), it seems that it creates the `enum` even when I specify (null, null). 
Fallbacking into the classic `switch` statement solves the problem.